src/main/java/duke/tasksmanager/Tasks.java

```diff
@@ -0,0 +1,32 @@
+package duke.tasksmanager;
+
+public class Tasks {
```

Comment:
This class name is plural, yet your attributes and method suggest that this could be for one class. You could take note of the naming conventions for collections v.s. singularity [here](https://se-education.org/guides/conventions/java/basic.html#layout).